composeCore/src/main/java/com/hamdan/forzenbook/compose/core/composables/Composables.kt

```diff
@@ -252,3 +271,90 @@ fun ForzenbookDialog(title: String, body: String, buttonText: String, onDismiss:
         modifier = Modifier.padding(ForzenbookTheme.dimens.mediumPad_1)
     )
 }
+
+@Composable
+fun PillToggleSwitch(
```

Comment:
These should be suffixed with `Res` for resource since they aren't images or strings